suite-common/test-utils/README.md

```diff
@@ -0,0 +1,25 @@
+# @suite-common/test-utils
+
+This package defines mocks for TrezorConnect to be shared between more packages that use these mocks in their tests.
+
+## Usage and configuration
+
+1. add `@suite-common/test-utils` to a project dependencies
+2. import mocks in your tests or fixtures
+
+```javascript
+import * as jestMocks from '@suite-common/test-utils';
```

Comment:
Please export it as one object, in the future we may want some other test utils here that we will want to export and it won't be only mocks.

So it should be exported as one object with name `mocks` or `testMocks`. Aka `import { testMocks } from '@suite-common/test-utils'`